src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Dynamically Import Images
+description: Learn how to dynamically import images using Vite's import.meta.glob function
+i18nReady: true
+type: recipe
+---
+import FileTree from '~/components/FileTree.astro'
+
+When working with custom components that include images, there will be times where you will need to dynamically import the image paths. But if the images are located in the `src` directory, it is possible to run into Astro or Vite import errors.  
```

Comment:
> But if the images are located in the `src` directory, it is possible to run into Astro or Vite import errors.

You can't (really) import from outside the `src` directory, so this sentence is a bit weird.